### ISSUE TITLE
Fhi Angular Highcharts: improve the default aspect ratios

### ISSUE THREAD

**Issue by proand** [folkehelseinstituttet/Fhi.Frontend.Demo#641]
### Description

End users of Fhi Angular Highcharts should get diagrams with height/width that works well on all devices independent of diagram type.
In issue #584 the solution for creating default aspect ratios in different viewport scenarios was updated, but it's still not good enough.

### Acceptance Criteria

1. Go to https://dev-designsystem.fhi.no/developer/components/highcharts
2. Test different diagramtypes on different viewport widths and see if the aspect ratios are ok.

**Comment by proand** [comment]
FYI @cathral

**Comment by dafn** [comment]
Is this still unconfirmed?

**Comment by proand** [comment]
No, we agreed to add it to ready on Monday, my bad for not updating status. Will do!
